### ISSUE TITLE
Connect to target issue

### ISSUE THREAD

**Issue by Jadmintor** [nil0x42/phpsploit#198]
im  sure for adding backdoor command to php, if i test using my own website this is work perfectly, but if i test to another web using same file php this is not connect to my phpsploit, please help

**Comment by nil0x42** [comment]
Obviously, in phpsploit you must use `set TARGET hxxp://TARGET/path/to/backdoor.php`, and then `exploit` to connect to your new target.

**Comment by Nader-abdi** [comment]
> im sure for adding backdoor command to php, if i test using my own website this is work perfectly, but if i test to another web using same file php this is not connect to my phpsploit, please help

You should check in the new target disabled functions php

**Comment by nil0x42** [comment]
@Nader-abdi it yould be a problem. Initially i wrote phpsploit to use only functions that are almost never disabled during the connection step.

If some functions i use on connection step tend to become disabled on some sites, it might be worth investigating, in order to update the connector.

Please tell me if you have information regarding this.

**Comment by Nader-abdi** [comment]
> @Nader-abdi it yould be a problem. Initially i wrote phpsploit to use only functions that are almost never disabled during the connection step.
> 
> If some functions i use on connection step tend to become disabled on some sites, it might be worth investigating, in order to update the connector.
> 
> Please tell me if you have information regarding this.

Yes I saw several problems when connecting , some target disabled this function ini_get_all() and  ini_set() when they modified the cream the problem was solved
